### PR TITLE
Add Input.Password

### PR DESCRIPTION
**Pull request by orzyyyy**
#13267
  
:worried:

### CODE REVIEW

**Comment by zombieJ** [comment]
Attach design preview from designer. Missing icon will be attached in antd icon:

**Comment by ycjcl868** [comment]
ci broken

**Comment by zy410419243** [comment]
I find that when I update snapshots locally, there is no `test-uuid` in my snap but in [`branch feature`](https://github.com/ant-design/ant-design/blob/feature/components/input/__tests__/__snapshots__/demo.test.js.snap#L50), and my branch is up to date. So what I can do now to pass ci...

**Comment by zombieJ** [comment]
@zy410419243 , rc-select update aria support. Remove all node_module and reinstall will help.
(PS: don't forget merge feature branch)

File: components/input/Password.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+export interface PasswordState {
+  visible: boolean,
+}
+
+const ActionMap: Record<string, string> = {
+  click: 'onClick',
+  hover: 'onMouseOver',
+}
+
+export default class Password extends React.Component<PasswordProps, any> {
```
Comment (ztplz) [inline components/input/Password.tsx]: `any` => `PasswordState`

**Review by ztplz — APPROVED** [review]

File: components/input/Password.tsx (hunk 1 above)
Comment (ztplz) [reply components/input/Password.tsx]: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/5c49d4ddc6f5bd06c5cf04d911398f3db05ead54/types/react/index.d.ts#L350
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/5c49d4ddc6f5bd06c5cf04d911398f3db05ead54/types/react/index.d.ts#L399

**Comment by zombieJ** [comment]
Icon comes! [`eye-invisible`](https://unpkg.com/@ant-design/icons@1.1.16/svg/fill/eye-invisible.svg) in @ant-design/icons - 1.1.16

**Comment by zy410419243** [comment]
> Icon comes! [`eye-invisible`](https://unpkg.com/@ant-design/icons@1.1.16/svg/fill/eye-invisible.svg) in @ant-design/icons - 1.1.16

Merge it into `feature` please.:horse:

**Comment by zombieJ** [comment]
@zy410419243 ,
Updated feature branch. Pls check:

**Comment by zy410419243** [comment]
Maybe need to update snapshots?

File: components/input/demo/password-input.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+密码框。
+
+## en-US
+
+Input type of password.
+
+````jsx
+  import { Input } from 'antd';
+
+  ReactDOM.render(<Input.Password placeholder="input password" />, mountNode);
```
Comment (afc163) [inline components/input/demo/password-input.md]: Don't indent here.

Comment (zy410419243) [reply components/input/demo/password-input.md]: done

File: components/input/index.en-US.md [hunk 3]
```diff
@@ -73,3 +73,7 @@ Supports all props of `Input`.
   <Input />
 </Input.Group>
 ```
+
+#### Input.Password
+
+`Input.Password` has no extra props.
```
Comment (afc163) [inline components/input/index.en-US.md]: We should add a prop `visibilityToggle={true|false}` to hide default toggle icon.

Comment (zy410419243) [reply components/input/index.en-US.md]: done

**Pull request merged**